media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored in the FLAC format.  flac123 is the only \
+component of flac-tools at this time."
+HOMEPAGE="http://flac-tools.sourceforge.net/"
+COPYRIGHT="2003-2017 Jake Angerman"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/flac-tools/flac123-$portVersion-release.tar.gz"
```

Comment:
I agree that we should use https whenever it is possible.
And I would even add that we should never give up using https just because some users have "unfriendly" proxys. BTW, As we know, doing so would help these bad proxys to remain unnoticed... and we don't want that ;-)

BTW, even on countries where people think they live in free democracies, when you get free wifi in some malls, what you have is spying proxys which do exactly what you mentioned: self-signed  certificates generated "on-the-fly". Stupid popular smartphones don't even notice it. Shame on them!